docs/releases/v2018.2.3.rst

```diff
@@ -0,0 +1,86 @@
+Gluon 2018.2.3
+==============
+
+Added hardware support
+~~~~~~~~~~~~~~~~~~~~~~
+
+ar71xx-generic
+^^^^^^^^^^^^^^
+
+* TP-Link
+
+  - CPE210 v3
+  
+ar71xx-nand
+^^^^^^^^^^^
+
+* Aerohive
+
+  - HiveAP 121
+
+mcp85xx-p1020
+^^^^^^^^^^^^^
+
+* Aerohive
+
+  - HiveAP 330
+
+ramips-mt76x8
+^^^^^^^^^^^^^
+
+* TP-Link
+
+  - TL-MR3420 v5 [#noibss]_
+  
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
+
+Bugfixes
+~~~~~~~~
+
+* fix for passwordless ssh when gluon-authorized-keys was used without gluon-setup-mode
+
+* fix for broken ingress traffic-shaping
+
+* fix packages for GL.iNet AR750 (relevant for IBSS mesh)
+
+* fix primary mac selection on TP-Link Archer C25 v1
+
+* fix for bootloader image generation for AVM Fritz!Box 4040
+
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to 4.9.188 or 4.14.137, depending on the target
+
+
+Known issues
+~~~~~~~~~~~~
+
+* Default TX power on many Ubiquiti devices is too high, correct offsets are
+  unknown (`#94 <https://github.com/freifunk-gluon/gluon/issues/94>`_)
+
+  Reducing the TX power in the Advanced Settings is recommended.
+
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is
+  disabled (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected
+  (like VMware when promiscuous mode is disallowed).
+
+* Inconsistent respondd API
+  (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API
+  will still be supported for a while.
+
+* Frequent reboots due to out-of-memory or high load due to memory pressure on
+  weak hardware specially in larger meshes
```

Comment:
especially

Reply:
> especially

this was in many release notes, not sure how to deal with it now...

Reply:
Fix it in this version, leave older versions untouched.

Reply:
done